### ISSUE TITLE
Build error when switching off unit test.

### ISSUE THREAD

**Issue by xkszltl** [Microsoft/onnxruntime#691]
Get the following errors if `-Donnxruntime_BUILD_UNIT_TESTS=OFF`.

```
  Finished generating code
  Finished generating code
  onnxruntime_pybind11_state.vcxproj -> C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\Release\onnxruntime_pybind11_state.pyd
EXEC : CMake error : cmake version 3.14.0 [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
  Usage: C:\Program Files\CMake\bin\cmake.exe -E <command> [arguments...]
  Available commands:
    capabilities              - Report capabilities built into cmake in JSON format
    chdir dir cmd [args...]   - run command in a given directory
    compare_files [--ignore-eol] file1 file2
                                - check if file1 is same as file2
    copy <file>... destination  - copy files to destination (either file or directory)
    copy_directory <dir>... destination   - copy content of <dir>... directories to 'destination' directory
    copy_if_different <file>... destination  - copy files if it has changed
    echo [<string>...]        - displays arguments as text
    echo_append [<string>...] - displays arguments as text but no new line
    env [--unset=NAME]... [NAME=VALUE]... COMMAND [ARG]...
                              - run command in a modified environment
    environment               - display the current environment
    make_directory <dir>...   - create parent and <dir> directories
    md5sum <file>...          - create MD5 checksum of files
    sha1sum <file>...         - create SHA1 checksum of files
    sha224sum <file>...       - create SHA224 checksum of files
    sha256sum <file>...       - create SHA256 checksum of files
    sha384sum <file>...       - create SHA384 checksum of files
    sha512sum <file>...       - create SHA512 checksum of files
    remove [-f] <file>...     - remove the file(s), use -f to force it
    remove_directory dir      - remove a directory and its contents
    rename oldname newname    - rename a file or directory (on one volume)
    server                    - start cmake in server mode
    sleep <number>...         - sleep for given number of seconds
    tar [cxt][vf][zjJ] file.tar [file/dir1 file/dir2 ...]
                              - create or extract a tar or zip archive
    time command [args...]    - run command and display elapsed time
    touch file                - touch a file.
    touch_nocreate file       - touch a file but do not create it.
    create_symlink old new    - create a symbolic link new -> old
  Available on Windows only:
    delete_regv key           - delete registry value
    env_vs8_wince sdkname     - displays a batch file which sets the environment for the provided Windows CE SDK installed in VS2005
    env_vs9_wince sdkname     - displays a batch file which sets the environment for the provided Windows CE SDK installed in VS2008
    write_regv key value      - write registry value

C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: The command "setlocal [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E make_directory  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E make_directory  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E make_directory  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E make_directory  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E make_directory  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/__init__.py  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/ThirdPartyNotices.txt  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/LICENSE  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/test/python/onnx_backend_test_series.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/test/python/onnxrunt
ime_test_python.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/test/python/onnxruntime_test_python_backend.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/test/python/onnxruntime_test_python_keras.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/test/python/onnxruntime_test_python_skl.py  [C:\Users\tolia\AppData\Local\Tem
p\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/backend/__init__.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/backend/backend.py C:/User
s/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/backend/backend_rep.py  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/__init__.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/_pybind_state.py C:/Users/tolia/Ap
pData/Local/Temp/onnxruntime/onnxruntime/python/onnxruntime_validation.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/session.py  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/build/Release/onnxruntime_pybind11_state.pyd  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxp
roj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/datasets/__init__.py  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/datasets/logreg_iris.onnx C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/datasets/mul_1.pb C:
/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/datasets/sigmoid.onnx  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/sklapi/__init__.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/sklapi/onnx_transformer.py
 [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/tools/__init__.py C:/Users/tolia/AppData/Local/Temp/onnxruntime/onnxruntime/python/tools/onnxruntime_test.py  [
C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmErrorLevel [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: exit /b %1 [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmDone [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: setlocal [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: "C:\Program Files\CMake\bin\cmake.exe" -E copy C:/Users/tolia/AppData/Local/Temp/onnxruntime/build/mklml/src/project_mklml/lib/mklml.dll C:/Users/tolia/AppData/Local/Temp/onnxruntime/build/mklml/src/project_mklml/lib/libiom
p5md.dll  [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmErrorLevel [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: exit /b %1 [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmDone [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :VCEnd" exited with code 1. [C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\onnxruntime_pybind11_state.vcxproj]
  onnxruntime.vcxproj -> C:\Users\tolia\AppData\Local\Temp\onnxruntime\build\Release\onnxruntime.dll
```

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Win10
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: 0.3.0
- Python version: 3.7
- GCC/Compiler version (if compiling from source): VS2017

**Comment by snnn** [comment]
Please turn it on. python binding(the onnxruntime_pybind11_state) depends on the unitest project(onnxruntime_test_all).

**Comment by snnn** [comment]
PR sent: #694

**Comment by snnn** [comment]
Hi @xkszltl , this problem should be solved by #694. Please check.
